src/app/gestao-usuarios/cadastro/individual/page.jsx

```
@@ -1,263 +1,287 @@
+				);
+				const autorizacoesUsuario = await atualizarAutorizacoes(
+					usuarioId,
+					autorizacoesIds,
+					session?.user?.access_token,
+				);
+
+				const novaLinha = {
+					id: uuidV4(),
+					mail: usuarioCadastrado.mail,
+					cpf: usuarioCadastrado.cpf,
+					nome: usuarioCadastrado["nome_usuario"],
```

Biome (1.9.4)

[error] The computed expression can be simplified without the use of a string literal.

Unsafe fix: Use a literal key instead.


(lint/complexity/useLiteralKeys)